usage/camp/src/test/java/org/apache/brooklyn/camp/brooklyn/catalog/CatalogYamlRebindTest.java

```diff
@@ -198,13 +191,7 @@ protected void runRebindWithCatalogAndApp(RebindWithCatalogTestMode mode) throws
                 addCatalogItems(String.format(locCatalogFormat, locVersion));
                 break;
             case STRIP_DEPRECATION_AND_ENABLEMENT_FROM_CATALOG_ITEM:
-                //Make sure that the flags are actually removed from the XML instead of still being there with false values
-                appItem.setDeprecated(true);
-                appItem.setDisabled(true);
-                mgmt().getCatalog().persist(appItem);
-                locItem.setDisabled(true);
-                locItem.setDeprecated(true);
-                mgmt().getCatalog().persist(locItem);
+                // nothing here -- but below we rebind with these fields removed to ensure that we can rebind
```

Comment:
Should keep these. They confirm that the disabled/deprecated elements are removed from the XML. Prior the adding them the test was successfully passing, but not testing anything because the elements weren't actually removed, so the default values of `false` remained.